mappings/net/minecraft/datafixer/fix/WeaponSmithChestLootTableFix.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_6858 net/minecraft/datafixer/fix/WeaponSmithChestLootTableFix
```

Comment:
This class name should be capitalized as `WeaponsmithChestLootTableFix` because the villager profession is registered as `minecraft:weaponsmith`, not `minecraft:weapon_smith`. The schema name is capitalized incorrectly.